tests/dragonfly/cluster_test.py

```diff
@@ -1177,11 +1177,6 @@ async def test_cluster_fuzzymigration(
     seeder = df_seeder_factory.create(keys=keys, port=nodes[0].instance.port, cluster_mode=True)
     await seeder.run(target_deviation=0.1)
 
-    fill_task = asyncio.create_task(seeder.run())
```

Comment:
why do you remove this?

Reply:
I don't see any point in having 2 calls for Seeder

Reply:
I didn't track the changes to this part of code, but there used to be a reason, likely now removed or just moved... Almost sure 😆

Reply:
Ok now I see that the seeder is not running while we are doing migration, right?
why so?

Reply:
we didn't do it previously too, during migration, we run counters.

Reply:
we don't control now when migration is finished so seeder can't be stopped before we migration is finished